docs/source/package.rst

```diff
@@ -0,0 +1,37 @@
+.. currentmodule:: torch.package
+
+torch.package
+=============
+
+API Reference
+-------------
```

Comment:
Can we add some sort of disclaimer that this is an experimental API, not publically released, etc.